delphi/GrFN/networks.py

```diff
@@ -506,7 +514,7 @@ def from_fortran_file(
             module_log_file_path,
             mod_mapper_dict,
             processing_modules,
-            save_intermediate_files = save_intermediate_files
+            save_file=save_file,
```

Comment:
If save_file is removed in the other two places, it should be removed here as well.